### PR TITLE
refactor: remove ContainerInfo from mgr layer, use ContainerMeta to replace it

### PR DESCRIPTION
**Pull request by skoo87**
Signed-off-by: skoo87 <marckywu@gmail.com>

**1.Describe what this PR did**
refactor: remove ContainerInfo from mgr layer, use ContainerMeta to replace it

**2.Does this pull request fix one issue?** 

**3.Describe how you did it**

**4.Describe how to verify it**

**5.Special notes for reviews**

### CODE REVIEW

**Comment by allencloud** [comment]
I found that ContainerInfo is not a type for API. You see:
`GET /containers/{id}/json` returns a type of `types.Container`
`GET /containers/json` returns a type of `[] types.ContainerJSON`
So no API side will use `ContainerInfo`, I think it is a legacy struct introduced into swagger.yml at the very beginning.

And now we already have ContanerInfo and ContainerMeta. I think we should remove one from mgr. Since they are the same.

**Comment by skoo87** [comment]
updated @allencloud

File: apis/server/container_bridge.go [hunk 1]
```diff
@@ -224,17 +224,17 @@ func (s *Server) getContainers(ctx context.Context, rw http.ResponseWriter, req
...
-		ID:      containerInfo.ID,
-		Name:    containerInfo.Name,
-		Image:   containerInfo.Config.Image,
-		Created: containerInfo.State.StartedAt,
-		State:   &types.ContainerState{Pid: containerInfo.State.Pid},
+		ID:      meta.ID,
+		Name:    meta.Name,
+		Image:   meta.Config.Image,
+		Created: meta.State.StartedAt,
+		State:   &types.ContainerState{Pid: meta.State.Pid},
```
Comment (allencloud) [inline apis/server/container_bridge.go]: We should add more data here for `ContainerState`.
Currently when we inspect a container, in `State`, there is only one PID, without any other state details.
Maybe in a follow-up.

Comment (skoo87) [reply apis/server/container_bridge.go]: Yes, do it in the follow-up pr.

**Comment by allencloud** [comment]
LGTM

**Pull request merged**